airflow/models/dag.py

```diff
@@ -300,8 +302,16 @@ def __init__(
         self.max_active_runs = max_active_runs
         self.dagrun_timeout = dagrun_timeout
         self.sla_miss_callback = sla_miss_callback
-        self._default_view = default_view
-        self.orientation = orientation
+        if default_view in DEFAULT_VIEW_PRESETS:
+            self.default_view = default_view
+        else:
+            raise AirflowException(f'Invalid values of dag.default_view: only support '
+                                   f'{DEFAULT_VIEW_PRESETS}, but get {default_view}')
```

Comment:
Do f-strings work in py3.6?

Reply:
Yeah, I think it work in py3.6 https://www.python.org/dev/peps/pep-0498/